Autopilot/SensorFusion/Src/SensorFusion.cpp

```diff
@@ -560,3 +551,44 @@ Airspeed_Data_t SF_GetRawAirspeed()
 
     return airspeedOutput;
 }
+
+Gps_Data_t SF_GetRawGPS()
+{
+    GpsData_t gpsData;
+    gpsObj->GetResult(gpsData);
+
+    Gps_Data_t gpsOutput;
+
+    gpsOutput.dataIsNew = gpsData.dataIsNew;
+    gpsOutput.ggaDataIsNew = gpsData.ggaDataIsNew;
+    gpsOutput.vtgDataIsNew = gpsData.vtgDataIsNew;
+    gpsOutput.latitude = gpsData.latitude;
+    gpsOutput.longitude = gpsData.longitude;
+    gpsOutput.utcTime = gpsData.utcTime;
+    gpsOutput.groundSpeed = gpsData.groundSpeed;
+    gpsOutput.altitude = gpsData.altitude;
+    gpsOutput.heading = gpsData.heading;
+    gpsOutput.numSatellites = gpsData.numSatellites;
+    gpsOutput.fixStatus = gpsData.fixStatus;
+    gpsOutput.sensorStatus = gpsData.sensorStatus;
+    gpsOutput.timeIsNew = gpsData.timeIsNew;
```

Comment:
Could use memcpy here:

https://en.cppreference.com/w/cpp/string/byte/memcpy

Same applies to other Get_SensorData functions